docker/vm/nodejs/nginx_vm.conf

```diff
@@ -0,0 +1,77 @@
+upstream django_app {
+  server django:8000 fail_timeout=0;
+}
+
+# Django server
+server {
+  listen 80;
+  server_name evalapi.example.com;
+
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+  location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+  }
+}
+
+server {
+  listen 443 ssl;
+  server_name evalapi.example.com;
+  client_max_body_size 400M;
+
+  ssl_certificate /etc/letsencrypt/live/example.com/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/example.com/privkey.pem;
+  include /etc/letsencrypt/options-ssl-nginx.conf;
+  ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
+
+  # Access Logs
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
+  location / {
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_pass http://django_app/;
+  }
+}
+
+# Nodejs
+server {
+  listen 80;
+  server_name example.com evalai.example.com;
+
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+  location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+  }
+}
+
+server {
+  listen 443 ssl;
+  sendfile on;
+  default_type application/octet-stream;
```

Comment:
please add appropriate comments over all the lines which have to be changed. Or add a setup.md

Reply:
Added commments